src/plugins/intel_cpu/src/nodes/roi_align.cpp

```diff
@@ -634,9 +635,9 @@ struct jit_uni_roi_align_kernel_f32 : public jit_uni_roi_align_kernel, public ji
 
 bool ROIAlign::isSupportedOperation(const std::shared_ptr<const ngraph::Node>& op, std::string& errorMessage) noexcept {
     try {
-        auto roiAlign = ngraph::as_type_ptr<const ngraph::opset3::ROIAlign>(op);
+        auto roiAlign = ngraph::as_type_ptr<const ngraph::opset9::ROIAlign>(op);
```

Comment:
make sure ROIAlign from all other opsets transformed to ROIAlign-9 if it can cover other version functionally. As I see there is no transformation done for ROIAlign-3 to ROIAlign-9. BTW, why do we need transformation ROIAlign-9 to ROIAlign-v0?

Reply:
we added 'ngraph::pass::ConvertROIAlign0To9'(based on previous roialign version definition, '0' means 'v3' here, if that confused you~) in the 'TransformationUpToCPUSpecificOpSet' of cpu plugin part.
we need transformation ROIAlign-9 to ROIAlign-0 because currently except cpu plugin other plugins have not support ROIAlign-9 yet, this transformation will make roialign op still worksn for them.

Reply:
yes, the function name is confusing, better to use same concept. if use version: 3-to-9, if use opsets: 0-to-1. what do u think?

Reply:
have used opset name style.